Automattic/woocommerce-payments
# Migrate DetailsLink component

Fixes part of #1351 

#### Changes proposed in this Pull Request

- Migrate `<DetailsLink />` and corresponding tests to TypeScript.
- Add declarations for the `<Link />` component from `@woocommerce/components`.
- Add declarations for the `<Gridicon />` component from `gridicons`.

#### Testing instructions

**Make sure to run `npm ci` and `npx webpack` (or `npm start`) before you start**

- Run `npm run test:js -- client/components/details-link` and make sure tests pass
- Open any of the tables that use the `<DetailsLink />` component (e.g. the deposits list) and make sure the `<DepositsLink />` renders correctly.

-------------------
